### DISCUSSION TITLE
Meaning of the code...

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [nvim-lualine/lualine.nvim#454] [Q&A] [answered]
```
normal = {
    a = { fg = colors.black, bg = colors.purple, gui = 'bold' },
    b = { fg = colors.purple, bg = colors.menu_grey },
    c = { fg = colors.comment_grey, bg = colors.black },
  },
```
what does the a, b and c means?  and there is inactive section... what does the mean anyways?

**Comment by fitrh** [comment, marked as answer]
```txt
+-------------------------------------------------+
| A | B | C                             X | Y | Z |
+-------------------------------------------------+
```

`a`, `b`, and `c` represent the left `section` of the status line, which is divided by 3 parts,
each `part` has their own fg, bg, and gui, each `part` has its own components, and if a
`component` do not specify their fg, bg, or gui, they will use the style of the `part` they belong to

The `inactive` define the state of each section for unfocused buffer

![screenshot_2021-11-26_203313](https://user-images.githubusercontent.com/55179750/143583010-b3325192-8dae-44de-b004-994bb971f14b.jpeg)

> The image above shows that `a = { fg = colors.black, bg = colors.red, gui = 'bold' }`
